core/src/main/java/com/xmartlabs/base/core/helper/ui/UiHelper.java

```diff
@@ -147,7 +148,8 @@ public static void validateFieldsAndPutErrorMessages(List<EditText> fields, bool
    * @param hideErrorLayout whether or not to enable the error on the given {@link EditText}
    * @return true if {@code editText} is valid using {@link #isValidField(EditText)}
    */
-  public static boolean validateField(EditText editText, boolean hideErrorLayout) {
+  public static boolean validateField(@NonNull EditText editText, boolean hideErrorLayout,
```

Comment:
Totally agree. The idea was to have a separate set of classes to handle all of these behavior. But, in the mean time, it should be fixed.
Thanks!